chipsec/modules/tools/smm/smm_ptr.py

```diff
@@ -36,6 +36,7 @@
     * ``config`` = use SMI configuration file <config_file>
     * ``fuzz`` = fuzz all SMI handlers with code in the range <smic_start:smic_end>
     * ``fuzzmore`` = fuzz mode + pass 2nd-order pointers within buffer to SMI handlers
+    * ``scan`` = fuzz mode + time measurement to identify SMIs that trigger long-running code paths
 - ``size``: size of the memory buffer (in Hex)
 - ``address``: physical address of memory buffer to pass in GP regs to SMI handlers (in Hex)
 
```

Comment:
Will you please add in the "Examples" section (after line 73:
` >>> chipsec_main.py -m tools.smm.smm_ptr -a scan 0x0:0xff`